templates/index.html

```diff
@@ -9,250 +9,266 @@
-					</h4>
-					<p>Monitors and tracks critical services to ensure proper opperations.</p>
-				</div>
-				<div class="four-col divider__col">
-					<h4 class="equal-height equal-height--mobile equal-height--align-middle">
-						<img src="{{ ASSET_SERVER_URL }}81ca0037-manage.svg" alt="Manage icon" class="no-margin">&nbsp;&nbsp;
-						<span>Manage</span>
-					</h4>
-					<p>Comes with a <a href="https://docs.ubuntu.com/maas/2.2/en/api" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" class="external">REST <abbr title="Application Programming Interface">API</abbr></a>, Web <abbr title="User Interface">UI</abbr> and <a href="https://docs.ubuntu.com/maas/2.2/en/manage-cli" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" class="external"><abbr title="Command Line Interface">CLI</abbr></a>.</p>
-				</div>
-			</div>
+						   			dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Fast deployment' });">Fast deployment&nbsp;&rsaquo;</a></h4>
+            </div>
+            <p>Zero-touch deployment of Ubuntu, CentOS, Windows, RHEL and SUSE.</p>
+            <p>Deploy linux distros in less than 5 minutes.</p>
+          </div>
+				</li>
+				<li class="p-matrix__item">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}db3bdf2d-machine-configuration.svg" alt="Machine configuration icon" />
+              <h4 class="p-heading-icon__title"><a class="p-link--strong p-heading--five" href="/tour#configure-servers" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about storage' }); dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Machine configuration' });">Machine configuration&nbsp;&rsaquo;</a></h4>
+            </div>
+            <p>Configure the machine’s network interfaces with bridges, <abbr title="Virtual Local Area Network">VLAN</abbr>s, bonds and more.</p>
+            <p>Create advanced filesystem layouts with <abbr title="Redundant array of independent disks">RAID</abbr>, <abbr title="Block Cache">bcache</abbr>, <abbr title="Logical Volume Management">LVM</abbr> and more.</p>
+          </div>
+				</li>
+				<li class="p-matrix__item">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}5626be9e-chassis-mangement.svg" alt="Chassis icon" />
+              <h4 class="p-heading-icon__title"><a class="p-link--strong p-heading--five" href="https://docs.ubuntu.com/maas/2.2/en/intel-rsd"
```

Comment:
Requires an ```aria-label``` to explain where the link takes you too. And that its an external link.